### PR TITLE
Refs #23960 -- Added note on how to generate absolute redirect URLs.

### PR DESCRIPTION
**Pull request by MoritzS**
Ticket: https://code.djangoproject.com/ticket/23960

### CODE REVIEW

File: docs/releases/1.9.txt [hunk 1]
```diff
@@ -794,6 +794,15 @@ no longer include the scheme and domain part of the URLs. For example,
 replaced by ``self.assertRedirects(response, '/some-url/')`` (unless the
 redirection specifically contained an absolute URL, of course).
 
+If you still need the old behavior, you can restore it by writing a :doc:`custom
```
Comment (timgraham) [inline docs/releases/1.9.txt]: Is it worth briefly describing your own use case?

Comment (MoritzS) [inline docs/releases/1.9.txt]: My use case is an ancient Apache version with mod_scgi that interprets relative redirects as "internal redirect". Do you think that is worth mentioning?

Comment (timgraham) [inline docs/releases/1.9.txt]: I think it wouldn't hurt to help explain that this isn't a common requirement.

**Comment by timgraham** [comment]
Updated and merged in 08b8c4697112e8dae90e72afc7d85bd31ead0410, thanks!

**Pull request closed without merging**